src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1420,10 +1416,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         {
             _terminal->SetBlockSelection(false);
             search.Select();
-            _renderer->TriggerSelection();
 
             // this is used for search,
-            // so hide the markers
+            // DO NOT call _updateSelectionUI() here.
+            // We don't want to show the markers so manually tell it to clear it.

```

Comment:
I personally find this comment a bit hard to understand. Why do we not want to show them?